src/extras/PMREMGenerator.js

```diff
@@ -104,6 +109,32 @@ PMREMGenerator.prototype = {
 
 	constructor: PMREMGenerator,
 
+	/**
+	 * Defines output and intermediary render target texture types.
+	 */
+	setDataType( type ) {
+
+		switch ( type ) {
+
+			case UnsignedByteType:
+				this._type = type;
+				this._format = RGBEFormat;
+				this._encoding = RGBEEncoding;
+				break;
+
+			case FloatType:
+			case HalfFloatType:
+				this._type = type;
```

Comment:
Here the `FloatType` input becomes also the output format, I believe?

Reply:
Nope, this is only when the user manually call `setDataType( FloatType / HalfFloatType )`.

Reply:
Oh, I think I understand why the confusion. `setDataType` is meant to set the framebuffers type ( and associated format/encoding ), not the input texture type.